doc/server.md

```diff
@@ -33,6 +33,7 @@ core.https\_allowed\_origin         | string    | global    | -         | -
 core.proxy\_https                   | string    | global    | -         | -                                 | https proxy to use, if any (falls back to HTTPS\_PROXY environment variable)
 core.proxy\_http                    | string    | global    | -         | -                                 | http proxy to use, if any (falls back to HTTP\_PROXY environment variable)
 core.proxy\_ignore\_hosts           | string    | global    | -         | -                                 | hosts which don't need the proxy for use (similar format to NO\_PROXY, e.g. 1.2.3.4,1.2.3.5, falls back to NO\_PROXY environment variable)
+core.trust\_ca\_certificates        | boolean   | global    | -         | -                                 | Whether to check for client certificates trusted by the CA
```

Comment:
The description probably should be clearer, something like `Whether to automatically trust clients signed by the CA`

Reply:
Done